### ISSUE TITLE
About pretraining

### ISSUE THREAD

**Issue by Itsanewday** [BAAI-DCAI/SegVol#19]
Thanks for your kindly sharing. Can you disclose more details about encoder pretraining?

**Comment by baifanxxx** [comment]
Hi,

I appreciate your interest in our work. We pre-train the encoder on large 3D CTs, and this data is released on huggingface and modelscope. 

Huggingface: https://huggingface.co/datasets/GoodBaiBai88/M3D-Cap
Huggingface: https://huggingface.co/datasets/GoodBaiBai88/M3D-Seg

Modelscope: https://www.modelscope.cn/datasets/GoodBaiBai88/M3D-Cap/summary
Modelscope: https://www.modelscope.cn/datasets/GoodBaiBai88/M3D-Seg/summary

In the training method, we use SimMIM and follow this repo (https://github.com/chenz53/MIM-Med3D). All data is prepared by normalization and diverse transforms. ViT is set with image size [32, 256, 256], patch size [4, 16, 16], masking ratio 0.6.

**Comment by Itsanewday** [comment]
Thanks very much, it really help me!

**Comment by Yuxin-Du-Lab** [comment]
These dataset links are all accessible now😊.
